src/averageAges.js

```diff
@@ -1,5 +1,6 @@
 'use strict';
 
+/* eslint-disable no-console */
```

Comment:
Consider removing the eslint-disable comment and fixing the linting errors instead. 🚨